### PR TITLE
10th temp commitment instanceuid only

### PR DESCRIPTION
**Pull request by red1408**
[#28] Modify storage commitment adapter to query only on sop instance uid (temporarily)

Needs 9th pull request.

### CODE REVIEW

File: import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/StorageCommitmentService.java [hunk 1]
```diff
@@ -149,8 +149,10 @@ public void run() {
             attrsItem.getString(Tag.ReferencedSOPClassUID));
         queryAttributes
             .setString(Tag.SOPInstanceUID, VR.UI, cmtItem.getInstanceUid());
-        queryAttributes
-            .setString(Tag.SOPClassUID, VR.UI, cmtItem.getClassUid());
+        // current (v1beta1) healthcare api wrongly returns
+        // "SOPClassUID is not a supported instance or study or series level attribute"
+        //queryAttributes
```
Comment (danielbeaudreau) [inline import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/StorageCommitmentService.java]: I'd prefer deleting the code for now, and we can use commit history to add it back once the issue is resolved on the server side.

Comment (red1408) [reply import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/StorageCommitmentService.java]: Ok

**Comment by danielbeaudreau** [comment]
Thanks, can you please rebase this as well?  (I guess all the rebasing is getting a bit problematic, but this PR doesn't remove the duplicate test script file, so it's necessary here anyway)

**Comment by red1408** [comment]
Done.

**Pull request merged**